.github/workflows/tests-e2e.yml

```diff
@@ -41,10 +41,10 @@ jobs:
         include:
           - browser: 'chrome'
             wp: 'latest'
-            coverage: true
+            snapshots: true
```

Comment:
@swissspidy Revert this change here

Reply:
Context:

I was looking at this workflow because I thought I'd needed it to change as well to prevent errors (turns out that wasn't needed).

When doing that, https://github.com/google/web-stories-wp/pull/5423#discussion_r530529914 came to mind.

IMHO here `snapshots` is clearer as it's about creating Percy snapshots, not about code coverage or anything.